addons/account/tests/test_account_invoice_report.py

```diff
@@ -114,8 +114,8 @@ def test_invoice_report_multiple_types(self):
         self.assertInvoiceReportValues([
             # amount_total  price_average   price_subtotal  residual    quantity
             [2000,          2000,           2000,           2000,       1],
-            [2000,          1000,           1000,           2000,       1],
-            [2000,          1000,           1000,           2000,       3],
+            [1000,          1000,           1000,           2000,       1],
```

Comment:
Indeed, we want the `amount_total` in the same currency than `price_subtotal`. In case of `residual`, better keep it in the invoice currency.